css-two-column-layout/styles.css

```diff
@@ -0,0 +1,80 @@
+
+.container {
+  display: flex;
+  flex-basis: auto;
+  flex-wrap: wrap;
+  margin-left: 10%;
+  margin-right: 10%;
+  font-family: 'Karla', sans-serif;
+  font-style: normal;
+  line-height: 1.5;
+}
+
+.row {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.top {
+  text-align: center;
+  margin-left: 15%;
+  margin-right: 15%;
+  margin-bottom: 25px;
+}
+
+h1 {
+  font-size: 50px;
+  width: 100%;
+  letter-spacing: 8px;
+}
+
+h1,
+h3 {
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+h3 {
+  margin-top: 25px;
+  text-align: left;
+}
+
+.pics {
+  width: 130px;
+  height: 130px;
+  float: left;
+  margin-bottom: 130px;
+}
+
+.normal-text {
+  font-weight: 400;
+}
+
+.column-half {
+  margin-left: auto;
+  margin-right: 100px;
```

Comment:
😱 😱 😱 😱 

That's a lot of margin! The class name also doesn't seem to correspond to the styling. If this is `column-half`, shouldn't it be something like `width: 50%`?